test/unittests/test_client.py

```
@@ -67,8 +67,49 @@ def test_collect_responses(self):
+
+    def test_on_collect_remove(self):
+        with patch("ovos_bus_client.client.client.load_message_bus_config") as mock_cfg:
+            mock_cfg.return_value = MagicMock(host="localhost", port=8181,
+                                              route="/core", ssl=False)
+            bus = MessageBusClient()
+        bus.client = MagicMock()
+        bus.connected_event.set()
+        bus.started_running = True
+
+        calls = []
+        handler = lambda cm: calls.append(cm)
```

Ruff (0.15.5)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `handler` as a `def`

(E731)